data/weapons-camo.ts

```diff
@@ -296,11 +589,41 @@ export const MELEE: Weapon[] = [
     name: 'KNIFE',
     type: WeaponType.MELEE,
     camo: {
-      fromBehind: 30,
-      crouching: 30,
+      kill: 50,
+      behind: 30,
+      crouch: 50,
+      noDeath2: 30,
       deadSilence: 50,
-      kills2NoDie: 30,
-      kills3NoDie: 30,
+      noDeath3: 30,
+    },
+  },
+];
+
+export const LAUNCHER: Weapon[] = [
+  {
+    id: 'fhj18',
+    name: 'FHJ-18',
+    type: WeaponType.LAUNCHER,
+    camo: {
+      streaks: 50,
+      UAV: 30,
+      tShield: 10,
```

Comment:
I think shield instead of tShield should be ok.

Reply:
there's both transform shield and turret shield so it might confuse people, as one is a scorestreak and the other is an operator skill

Reply:
ooh, good call, but `t` in the `tShield` can also be turret. Maybe name it to `transformShield`